android/clientlib-ktx/build.gradle

```diff
@@ -0,0 +1,42 @@
+plugins {
+    id 'com.android.library'
+    id 'org.jetbrains.kotlin.android'
+}
+
+android {
+    compileSdk 32
+
+    defaultConfig {
+        minSdk 19
```

Comment:
This is less than `clientlib` - let's standardize on `23`

Reply:
You won't get an argument from me!